src/main/kotlin/no/kartverket/matrikkel/bygning/matrikkel/adapters/KodeMappers.kt

```diff
@@ -0,0 +1,46 @@
+package no.kartverket.matrikkel.bygning.matrikkel.adapters
+
+import no.kartverket.matrikkel.bygning.models.kodelister.AvlopKode
+import no.kartverket.matrikkel.bygning.models.kodelister.EnergikildeKode
+import no.kartverket.matrikkel.bygning.models.kodelister.OppvarmingKode
+import no.kartverket.matrikkel.bygning.models.kodelister.VannforsyningKode
+import no.statkart.matrikkel.matrikkelapi.wsapi.v1.domain.bygning.koder.AvlopsKodeId
+import no.statkart.matrikkel.matrikkelapi.wsapi.v1.domain.bygning.koder.EnergikildeKodeId
+import no.statkart.matrikkel.matrikkelapi.wsapi.v1.domain.bygning.koder.OppvarmingsKodeId
+import no.statkart.matrikkel.matrikkelapi.wsapi.v1.domain.bygning.koder.VannforsyningsKodeId
+
+fun mapEnergikilde(kodeId: EnergikildeKodeId): EnergikildeKode = when (kodeId.value) {
+    0L -> EnergikildeKode.Elektrisitet
+    1L -> EnergikildeKode.OljeParafin
+    2L -> EnergikildeKode.Biobrensel
+    3L -> EnergikildeKode.Solenergi
+    4L -> EnergikildeKode.Varmepumpe
+    5L -> EnergikildeKode.Gass
+    6L -> EnergikildeKode.Fjernvarme
+    7L -> EnergikildeKode.AnnenEnergikilde
+    else -> throw RuntimeException("Ukjent energikildekode: ${kodeId.value}")
```

Comment:
For å få en litt mer presis exception-type.

```suggestion
    else -> throw IllegalArgumentException("Ukjent energikildekode: ${kodeId.value}")
```

Reply:
Jeg kan godt gjøre det, men jeg gadd ikke "lete" etter en mer nøyaktig exception-type før vi hadde bestemt hvordan slike feil skal meldes generelt.

Reply:
Ja, enig i det. Vi må ta en prat om det. Tenkte bare at akkurat denne så fantes det en fin java-type for det.
Men kan godt la det være så tar vi en generell prat om det.